src/components/cms/SkillCardScrollList/SkillCard.js

```diff
@@ -20,7 +20,7 @@ import {
   Image,
   TitleContainer,
   RatingContainer,
-  TotalRating,
+  AverageRating,
```

Comment:
Please make changes related to the issue only

Reply:
@anishagg17 I will correct it but now the site is not running on the localhost when I reforked the repository.
![image](https://user-images.githubusercontent.com/44136704/71961267-022a7a00-321d-11ea-879b-ffc9642db7e2.png)
How should I solve this.

Reply:
Ya the server is not working well since 2 days thats why you are facing these errors because of cors parameter

It 's not because you reforked the repository.

Reply:
@anishagg17 Ohh Okay ! When the error would get rectified?